### PR TITLE
just-js: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.1.13

### CODE REVIEW

**Comment by navarroaxel** [comment]
Should we rename this to `just-js`?

**Comment by mfrw** [comment]
> Should we rename this to `just-js`?

Yes :) Please

**Comment by kbdharun** [comment]
> Should we rename this to `just-js`?

Yes, +1

File: pages/linux/just-js.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# just
+
+> A V8 JavaScript runtime for Linux only.
```
Comment (kbdharun) [inline pages/linux/just-js.md:3]:
```suggestion
> A V8 JavaScript runtime for Linux.
```
I think mentioning Linux alone is self-sufficient here.

**Review by kbdharun — APPROVED** [review]
LGTM, Just a small suggestion.

**Review by kbdharun — APPROVED** [review]
Thanks

**Review by blueskyson — APPROVED** [review]
Maybe we have to deal with different commands with the same name later.

**Comment by pixelcmtd** [comment]
@blueskyson We should. There are more examples of this than you might think: Very recently, I discovered that SpiderMonkey's REPL is also called `sm`, in a not-so-fun way.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**